workflow/log-monitoring/search-filter.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Search & Filter Your Logs
+sidebarTitle: Search & Filter
+---
+
+The Search and Filter features within the Logs page provide a streamlined method for accessing your data. Create persistent filters to preserve your customized views, efficiently browse through your logs, and access important collections of logs in a single button click.
+
+To quickly filter your log data, use the leftmost column on the logs page to toggle on and off filters sorted by __Levels__, __Additional__, __NameSpace__, __APM Projects__, and __Sources__.
+
+# Create New Filters
+
+Filters allow you to easily access data without redundantly filtering through irrelevant logs. Create a custom filter that you have access to with a single click.
```

Comment:
In changing this from "Create Custom" to "Save Filter..." this language also needs to change. I'd also recommend replacing the "Create a custom filter you have access to with a single click." as it's a filler / fluff sentence we've talked about. Recommend adding something that expands on what this section is or exactly what value users get from using it.